Source/Core/Core/NetPlayQoS.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+struct _ENetPeer;
+typedef ENetPeer _ENetPeer;
+
+class NetPlayQoSSession
+{
+public:
+  NetPlayQoSSession(ENetPeer* peer);
+  ~NetPlayQoSSession();
+
+  inline bool Successful() const { return m_success; }
```

Comment:
You don't need to specify `inline` here. Functions defined inside a class definition are already inline by default.